src/libraries/Microsoft.Extensions.Hosting/src/Host.cs

```diff
@@ -52,85 +49,11 @@ public static class Host
         ///   </list>
         /// </remarks>
         /// <param name="args">The command line args.</param>
-        /// <returns>The initialized <see cref="IHostBuilder"/>.</returns>
+        /// <returns>The configured <see cref="IHostBuilder"/>.</returns>
```

Comment:
I was thinking that it's not just initialized anymore, it's configured with defaults - so I just thought it was configured. I suppose initialized is fine, I could revert that if you'd like?

Reply:
The refactoring doesn't change anything about what this method does. It was configured before as well.

I guess to me, since this is called `CreateDefaultBuilder`, it makes sense to say `initialized`. It also matches the docs on the above overload

https://github.com/dotnet/runtime/blob/82ca681cbac89d813a3ce397e0c665e6c051ed67/src/libraries/Microsoft.Extensions.Hosting/src/Host.cs#L33-L35

Reply:
That makes sense and I agree - reverted, thanks you.